src/components/ExpandableListItemActions.tsx

```diff
@@ -22,11 +22,13 @@ export default function ExpandableListItemActions({ children }: Props): ReactEle
   if (Array.isArray(children)) {
     return (
       <Grid container direction="row">
-        {children.map((a, i) => (
-          <Grid key={i} className={classes.action}>
-            {a}
-          </Grid>
-        ))}
+        {children
+          .filter(x => x)
```

Comment:
what does this do? This eliminates some undefined elements? (so `children` is array with holes?) Can you elaborate a bit how that can happen?